src/main/rx/spektrum.c

```diff
@@ -130,6 +132,51 @@ uint8_t spektrumFrameStatus(void)
 
     rcFrameComplete = false;
 
+	//Check for bind type in the first packet after startup. This is provided in the second byte in a data packet from the internal remote.
+    //This packet wont contain any data if the remote was bound using a separate receiver (bound as external remote)
+    if(!initialPacket){
+
+        switch (spekFrame[1]){
+            case 0xb2:       //11MS 2048 DSMX
+                rxConfig->serialrx_provider = SERIALRX_SPEKTRUM2048;
+                saveConfigAndNotify();
```

Comment:
I think save this one's if it is not properly set is not a problem. You can check the actual setting and update if needed. This will only write once or after an RX protocoll change.